docs/data.md

```diff
@@ -56,7 +56,7 @@ Additionally, the OSV.dev team maintains a conversion pipeline for:
 
 -   [Debian Security Advisories](https://storage.googleapis.com/debian-osv/index.html),
     using the conversion tools
-    [here](https://github.com/ossf/osv-schema/tree/main/tools/debian).
+    [here](https://github.com/google/osv.dev/tree/master/vulnfeed/tools/debian).
```

Comment:
I got 404 -page not found for this URL?

Reply:
:facepalm: Didn't press save after adding the s to vulnfeed(s)

Reply:
Fixed